cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -126,6 +126,10 @@ def _get_org(self, name):
             org.global_org = True
         return org
 
+    @property
+    def project_cache_dir(self):
+        return self.project_config.project_cache_dir
```

Comment:
What about other keychains? Is it intentional for code that uses `keychain.project_cache_dir` to break if running with EnvironmentProjectKeychain or a keychain that stores org configs as Django models?

Reply:
What do you think about if get_orginfo_cache_dir had this:

```
    def get_orginfo_cache_dir(self):
        assert self.keychain, "Keychain should be set"
        if self.global_org or not hasattr(self.keychain, "project_cache_dir"):
            config_dir = self.keychain.config_local_dir
        else:
            config_dir = self.keychain.project_cache_dir

        cache_dir = config_dir / "orginfo" / self.get_domain()
```

Reply:
@prescod So the proposal is to store in the global location (~/.cumulusci) if the keychain doesn't support a project-specific cache?

I think that makes sense. But what about giving BaseProjectKeychain an implementation of `project_cache_dir` that simply returns config_local_dir, and then we don't need this hasattr check?

Btw, do you think we should we use the pyfilesystem API for interactions with the cache, so that it's easier to switch to alternate storage if we want something that can be made to survive across resumed builds?

Reply:
With respect to project_cache_dir, I did that in c2429df

With respect to PyFilesystem. Yeah, I guess that makes sense. I'll try to do that in the next several days.